api/synth_trans.md

```diff
@@ -0,0 +1,11 @@
+# `synthesis-service.` and `translate-service.` APIs
+
+These retrieve the speech synthesis and translation results for their respective extensions.
+
+### `GET translate-service.scratch.mit.edu/translate?language=<language code>&text=<text>`
+
+Retrieves the translation of the input text in the given language. Assumed to use Google Translate
```

Comment:
I found [this citation](https://scratch.mit.edu/discuss/post/3778811/) for the translate extension using Google Translate. It's from four years ago (almost to the day!), but that's also a while after 3.0 released, so it's possibly still accurate. Maybe we could add something like this?

```suggestion
Retrieves the translation of the input text in the given language. Uses Google Translate [as of Dec. 9, 2019](https://scratch.mit.edu/discuss/post/3778811).
```

Reply:
Added 👍 (idk what else to say lol)